docs/competition/team-support/battery-charging.md

```diff
@@ -7,44 +7,76 @@ original:
 
-Teams will bring discharged batteries to the Battery Charging station and require a replacement charged battery. A Battery Charging volunteer should receive the discharged battery from the team and place it in the discharged battery box. A charged battery should be removed from the charged battery box and handed to the team. Do not allow teams to take batteries from the box themselves - we want to prevent teams from hoarding batteries.
+### Charging a discharged battery - Battery Charging Station
 
-### Charging a discharged battery
+All of the battery chargers will be set-up and ready to use. Before a battery is connected the screen must display `LiPo BALANCE 2.2A 11.1V(3S)`. If it displays anything else then please do not use it and ask Helpdesk to summon help.
 
-All of the battery chargers will be set-up and ready to use. Before a battery is connected the screen must display `LiPo BALANCE 2.2A 11.1V(3S)`. If it displays anything else then please do not use it and ask [Helpdesk](/competition/team-support/helpdesk) to summon Rich Barlow. Batteries are charged as follows:
+Batteries are charged as follows:
 
- 1. Check that the battery is not physically damaged. There should be no splits or tears in the body of the battery, no nicks in the wire insulation, no exposed wire and the battery should not be so swollen that it is no longer squidgy. There will be a couple of example batteries at the Battery Charging station to compare to.
- 1. If the battery is damaged, tape over the yellow XT60 connector with red tape and place it in the damaged battery box.
- 1. If the battery is not damaged, plug the yellow XT60 connector into the charger cable and the white balance connector into the 4-pin socket on the side of the charger.
- 1. Place the battery in the charging bag next to the charger.
- 1. Hold down the Start/Enter button for about 3 seconds until the screen displays `R:3SER S:3SER CONFIRM(ENTER)`. If both numbers are not 3 then disconnect the battery and summon Rich Barlow via Helpdesk.
- 1. If the display is as expected, press Start/Enter to commence charging.
+- Check that the battery is not physically damaged. There should be no splits or tears in the body of the battery, no nicks in the wire insulation, no exposed wire and the battery should not be so swollen that it is no longer squidgy. There will be a couple of example batteries at the Battery Charging station to compare to.
+- If the battery is damaged, tape over the yellow XT60 connector with red tape and place it in the damaged battery box.
+- If the battery is not damaged, plug the yellow XT60 connector into the charger cable and the white balance connector into the 4-pin socket on the side of the charger.
+- Place the battery in the charging bag next to the charger.
+- Follow the instructions as found in the [Student Robotics kit docs](https://www.studentrobotics.org/docs/kit/batteries/).
```

Comment:
I believe in the past we have tried to use only a single type of charger on the battery charging station, in order to simplify this safety-important role.

Whether or not we diverge from that, I think it's important that the document which is present on the charging station (whether that be an exact printout of this page or something built from it) be a complete document rather than one which links to external sources. We should not assume that volunteers have any access to other resources.